inst/qmongr.yml

```diff
@@ -14,7 +14,14 @@ data:
     column:
         year: "Aar"
         qi_id: "KvalIndID"
-        unit: "ShNavn"
-        unit_id: "OrgNrShus"
+        unit_name:
+            sh: "SykehusNavn"
+            hf: "HF"
+            rhf: "RHF"
+            hfshort: "Hfkortnavn"
```

Comment:
Is it possible to replace `hf: "HF"` with `hf: "Hfkortnavn"` and remove `hfshort: "Hfkortnavn"`?

Reply:
that is a great idea!!!